### PR TITLE
[MMI] Adds mmi build type + manifest files and images

### PR DESCRIPTION
**Pull request by zone-live**
## Explanation

As part of the MetaMerge initiative we need to have the `mmi` build type to be able to run the extension with the MMI flavour, this PR adds that initial logic for MMI build type, manifest json files and the necessary images for MMI. Others will follow after.

Jira ticket ref: https://consensyssoftware.atlassian.net/browse/MMI-2481

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving

## Pre-merge reviewer checklist

- [x] PR is linked to the appropriate GitHub issue

### CODE REVIEW

**Comment by zone-live** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by zone-live** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by zone-live** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by albertolive — DISMISSED** [review]

File: development/build/mmi-scripts/build-for-all-custodians.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable node/no-extraneous-require */
+const { spawn } = require('child_process');
+
+const CONFIG_URL =
+  'https://mmi-configuration-api.codefi.network/v1/configuration';
```
Comment (legobeat) [inline development/build/mmi-scripts/build-for-all-custodians.js:6]: This url is currently 404ing - is there a sample to look at?

**Comment by Sylvia24** [comment]
We'd be happy to look into this for you. kindly file a support ticket below 
https://forms.gle/Wmpm6NLzYm9uXYuU9

File: development/build/mmi-scripts/build-for-all-custodians.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (mcmire) [inline development/build/mmi-scripts/build-for-all-custodians.js:1]: We are slowly converting the codebase to TypeScript so if it is possible for this to be in TypeScript that would be appreciated! Note that you will want to run this script using `yarn ts-node`. You can save from having to remember this by adding a shebang at the top (something like `#!yarn ts-node`) and making this file executable (`chmod +x development/build/mmi-scripts/build-for-all-custodians.js`). Then you can  just run this file by saving `development/build/mmi-scripts/build-for-all-custodians.ts`.

Comment (zone-live) [reply development/build/mmi-scripts/build-for-all-custodians.js:1]: Thanks @mcmire, makes sense, let us update the file and I'll get back to you! 💪🏼

**Review by brad-decker — DISMISSED** [review]
LGTM.

File: app/build-types/mmi/manifest/_base.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "16": "images/icon-16.png",
+    "19": "images/icon-19.png",
+    "32": "images/icon-32.png",
+    "38": "images/icon-38.png",
+    "48": "images/icon-48.png",
+    "64": "images/icon-64.png",
+    "128": "images/icon-128.png",
+    "512": "images/icon-512.png"
+  },
+  "name": "__MSG_appNameMmi__",
```
Comment (Gudahtt) [inline app/build-types/mmi/manifest/_base.json:24]: The message `appNameMmi` seems to be missing from our localizations. See the `appNameBeta` message for an example of what we need here (we'll want it to be present in every single locale file, and we don't need it to be translated).

Chrome will reject any extension as invalid if a referenced message is missing in any locale.

**Review by Gudahtt — COMMENTED** [review]
Looks good aside from the missing localized message!

File: app/build-types/mmi/manifest/_base.json (hunk 3 above)
Comment (zone-live) [reply app/build-types/mmi/manifest/_base.json:24]: Oh! ok, we thought we could add them all later in a new/specific PR, ok we will have to add them as we go then 👍🏼 thanks!

**Review by brad-decker — DISMISSED** [review]

**Review by Gudahtt — APPROVED** [review]
LGTM! Normally we'd need this across all locales, but I'm told that MMI is only released for `en` so this should be sufficient.

We should investigate how to remove these app names from the localized message files altogether, and inject it at build-time instead. That would ensure that build-specific messages aren't included in builds they aren't needed in, and it would save our translators from getting confused by this and trying to translate it unnecessarily.

**Review by albertolive — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Pull request merged**